ereolen_support/ereolen_support.helper.inc

```diff
@@ -0,0 +1,255 @@
+<?php
+
+/**
```

Comment:
General for this file: 

* Missing user input validation to protect jira
* Missing responce json validation
* Missing http remote server request error handling and debugging information on failure